xml/cl.xml

```diff
@@ -7246,5 +7268,34 @@ server's OpenCL/api-docs repository.
                 <enum name="CL_MEM_DEVICE_ID_INTEL"/>
             </require>
         </extension>
+        <extension name="cl_khr_command_buffer_multi_device" requires="cl_khr_command_buffer" comment="in sync with rev 16" supported="opencl">
```

Comment:
niptick, since revisions have been squashed
```suggestion
        <extension name="cl_khr_command_buffer_multi_device" requires="cl_khr_command_buffer" comment="in sync with version 0.9.0" supported="opencl">
```